tools/commands.js

```diff
@@ -392,6 +392,18 @@ function doRunCommand (options) {
     mobileServer = mobileServer + ":" + parsedMobileServer.port;
   }
 
+  let errorAppPath = files.pathJoin(files.convertToStandardPath(__dirname),
```

Comment:
Is there any reason this is `let` and not `const`? https://github.com/meteor/javascript#13.1